src/commands/Minion/slayershop.ts

```diff
@@ -0,0 +1,111 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { BotCommand } from '../../lib/BotCommand';
+import { stringMatches } from '../../lib/util';
+import { Time } from 'oldschooljs/dist/constants';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import { Monsters } from 'oldschooljs';
+
+const slayerShopItems = [
+	{
+		name: 'Aviansie',
+		slayerPointsRequired: 80,
+		ID: Monsters.AberrantSpectre.id
+	},
+	{
+		name: 'Basilisk',
+		slayerPointsRequired: 80,
+		ID: Monsters.Basilisk.id
+	},
+	{
+		name: 'Boss',
+		slayerPointsRequired: 200,
+		ID: Monsters.Vorkath.id
+	},
+	{
+		name: 'Lizardman',
+		slayerPointsRequired: 75,
+		ID: Monsters.Lizardman.id
+	},
+	{
+		name: 'Mithril dragon',
+		slayerPointsRequired: 80,
+		ID: Monsters.MithrilDragon.id
+	},
+	{
+		name: 'Red dragon',
+		slayerPointsRequired: 50,
+		ID: Monsters.RedDragon.id
+	}
+	/*
+	{
+		name: 'TzHaar',
+		slayerPointsRequired: 100,
+		ID: Monsters.TzHaarKet.id
+	}
+	*/
+];
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			usage: '<unlockname:...string>',
+			usageDelim: ' ',
+			oneAtTime: true,
+			cooldown: 1,
+			altProtection: true
+		});
+	}
+
+	async run(msg: KlasaMessage, [unlockname = '']: [string]) {
+		await msg.author.settings.sync(true);
+		msg.author.slayerInfo.slayerPoints;
```

Comment:
what is this for?

`const { slayerInfo } = msg.author;`

refactor this file to use `slayerInfo` instead of `msg.author.slayerInfo`